server/src/i18n/en.json

```diff
@@ -33,5 +33,8 @@
 	},
 	"general": {
 		"home": "Please check /api routes."
+	},
+	"mail": {
+		"activation": "Hello, please check below the activation link: %s "
```

Comment:
Don't forget to write it in fr.json as well (even if is with the same message (or we can do this at the end to all languages)

Reply:
done